### PR TITLE
Playground E2E tests

### PR DESCRIPTION
**Pull request by james-iohk**
Plutus Playground E2E tests using [Playwright](https://playwright.dev/).

CI is configured to run two sanity tests on three environments ([production](https://playground.plutus.iohkdev.io/), [staging](https://plutus-playground-plutus-apps-staging.plutus.aws.iohkdev.io/), and local instance) with two different browsers (Chrome and Firefox). Html report is produced and uploaded as artefact.

Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: plutus-playground-client/e2e-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Playwright E2E Tests
+
+1. `nvm use`
+2. `npm install -D @playwright/test`
```
Comment (koslambrou) [inline plutus-playground-client/e2e-tests/README.md:4]: Change this to `npm i`

File: plutus-playground-client/e2e-tests/playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    },
+/*    {
+      name: 'safari',
+      use: {
+        browserName: 'webkit',
+        viewport: { width: 1280, height: 720 },
+      }
+    }*/
+  ],
+};
```
Comment (koslambrou) [inline plutus-playground-client/e2e-tests/playwright.config.ts:46]: Why is that commented? Remove it if not necessary or add a comment on why this is commented.

**Review by koslambrou — APPROVED** [review]

File: plutus-playground-client/e2e-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## Playwright E2E Tests
+
+1. `nvm use`
```
Comment (koslambrou) [inline plutus-playground-client/e2e-tests/README.md:3]: Specify that this is optional if your using `nix-shell`

File: plutus-playground-client/e2e-tests/playwright.config.ts (hunk 2 above)
Comment (james-iohk) [reply plutus-playground-client/e2e-tests/playwright.config.ts]: removed

**Pull request merged**